dynawo/sources/Common/CMakeLists.txt

```diff
@@ -59,104 +59,95 @@ set(COMMON_SOURCES
   DYNBitMask.cpp
   DYNCommon.cpp
   DYNError.cpp
-  DYNErrorQueue.cpp
   DYNTerminate.cpp
   DYNExecUtils.cpp
   DYNEnumUtils.cpp
   DYNFileSystemUtils.cpp
-  DYNTrace.cpp
-  DYNTraceStream.cpp
   DYNMessage.cpp
   DYNMessageTimeline.cpp
-  DYNIoDico.cpp
   DYNGraph.cpp
   DYNParameter.cpp
   DYNSparseMatrix.cpp
   ${CPP_KEYS}
+  DYNErrorQueue.cpp
+  DYNIoDico.cpp
+  DYNTimer.cpp
+  DYNTrace.cpp
+  DYNTraceStream.cpp
   )
 
 set(COMMON_INCLUDE_HEADERS
   DYNBitMask.h
   DYNCommon.h
   DYNError.h
-  DYNErrorQueue.h
   DYNTerminate.h
   DYNExecUtils.h
   DYNEnumUtils.h
   DYNFileSystemUtils.h
-  DYNTrace.h
-  DYNTraceStream.h
   DYNGraph.h
   DYNMessage.h
   DYNMessage.hpp
   DYNMessageTimeline.h
   DYNMessageTimeline.hpp
   DYNMacrosMessage.h
-  DYNIoDico.h
   DYNParameter.h
   DYNSparseMatrix.h
   DYNParameter.hpp
   gtest_dynawo.h
   ${INCLUDE_KEYS}
-  )
-
-set(UTIL_SOURCES
-  DYNTimer.cpp
-  )
-
-set(UTIL_INCLUDE_HEADERS
+  DYNErrorQueue.h
+  DYNIoDico.h
   DYNTimer.h
```

Comment:
Why are we removing the Dynawo_util library?

Reply:
With the current implementation it is useless, it will be reintroduced with the changes for the singleton but it easier to delete it for the time being as it should never as existed after windows port